ecommerce/HooksHandler.php

```diff
@@ -556,5 +562,72 @@ public function store_billing_address_for_order( array $order_data ) {
 			$meta_value
 		);
 	}
-}
 
+
+	/**
+	 * Recalculate and update order data before manual "Pay Now" procession.
+	 *
+	 * @since 3.9.2
+	 *
+	 * @param object $order_data    The order data object.
+	 * @param object $checkout_data The checkout data object.
+	 *
+	 * @return array Updated order data array with recalculated prices, discounts, and taxes.
+	 */
+	public function manual_pay_now_order_update( object $order_data, object $checkout_data ) {
```

Comment:
No need this hook callback method. Reuse order controllers `create` method's logic